cli.js

```diff
@@ -142,7 +142,7 @@ function run(data) {
             if (err) {
                 error(err);
             } else {
-                process.stdout.write(val);
+                process.stdout.write(val, process.exit);
```

Comment:
I'm sure there was some reason i put it there even if i can't get it right now as this doesn't have anything todo with the PRs main purpose. Do you have any insights on this being a bad practice?